content/posts/mongo_db_databases.md

```diff
@@ -0,0 +1,58 @@
+---
+title: "Introducing MongoDB databases"
+date: 2024-10-14
+tags:
+    - News
+author: Costin Sin
+linkedIn: https://www.linkedin.com/in/costin-sin/
+thumbnail: /images/ferretdb-genezio.webp
+preview: Genezio users can now set up MongoDB databases! You can now easily manage your MongoDB databases alongside your serverless functions. Learn more about this new feature.
+# meta data start
+description: Learn about the new MongoDB database feature in Genezio. Easily set up and manage MongoDB databases alongside your serverless functions.
+meta_og_url: "https://genezio.com/blog/mongo_db_databases/"
+meta_og_image: "https://genezio.com/images/ferretdb-genezio.webp"
+# meta data end
+customHeader: "White header"
+customFooter: "White footer"
+readTime: 10
+url: /blog/mongo_db_databases/
+---
+
+Hello everyone! We’re excited to share some big news: Genezio users can now set up MongoDB databases! Thanks to our new partnership with MongoDB Atlas, you can easily manage your MongoDB databases alongside your serverless functions. This means you get a powerful, fully managed database service that’s easy to use, scalable, and secure.
+
+## What Does This Mean for You?
+
+If you’re a Genezio user, you already know how simple it is to deploy serverless functions with just a few clicks. Now, with MongoDB hosting built into our platform, you can also quickly set up a NoSQL database to support your microservices, APIs, and apps. Here’s how this feature can improve your workflow:
+
+1. **Easy Integration**: With Genezio and MongoDB Atlas working together, you won’t have to deal with the hassle of setting up and managing a database. Whether your project is small or large, you can deploy MongoDB databases right from Genezio’s interface with just a few clicks.
+
+2. **Secure and Reliable**: Security is a top priority, especially when handling important data. MongoDB Atlas protects your information with top-notch security features like encryption, fine-grained access controls, and built-in backups. This keeps your data safe and always available.
+
+## How to Get Started
+
```

Comment:
Ready to give it a try? Here’s how to get started:

1. **Log in to your Genezio account**: Head over to the [Databases section] (https://app.genez.io/databases?utm_source=genezio&utm_medium=blog&utm_campaign=mongodb), where you can now create and manage your MongoDB databases.
2. **Create a new MongoDB instance**: Just click to create a new MongoDB database. Once it’s set up, you’ll get a connection string that you can use to link your database to your app. 
3. **Integrate with your project**: Use the connection string to start querying your database from your app or API. You can also add your database details to your YAML configuration file if you prefer managing things via code.